### ISSUE TITLE
Remove/resolve remaining "UNREVIEWED" mappings (12)

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/european-union-support#157]
3 are also "REPEAT".

**Comment by odscjen** [comment]
Only 3 left now waiting on resolution of https://github.com/open-contracting/european-union-support/issues/83#issuecomment-1387442505 and https://github.com/open-contracting/european-union-support/issues/102#issuecomment-1387356894

**Comment by odscjen** [comment]
The only remaining UNREVIEWED is OPP-080 which is covered by https://github.com/open-contracting/european-union-support/issues/158#issuecomment-1404116926

**Comment by jpmckinney** [comment]
Closing in favor of #158.
